common/reviews/api/heft.api.md

```diff
@@ -4,6 +4,7 @@
 
 ```ts
 
+import { AsyncParallelHook } from 'tapable';
```

Comment:
I feel like this API surface should be split up apart into several NPM packages. But let's defer that design work until we can see a working prototype that builds a real world project.